src/runtime/driver/op/mod.rs

```diff
@@ -99,7 +99,7 @@ where
     T: Completable,
 {
     /// Create a new operation
-    fn new(data: T, inner: &mut driver::Driver) -> Self {
+    fn new(data: T, inner: &mut runtime::driver::Driver) -> Self {
```

Comment:
Not a comment on this reorg. I wonder why this parameter is named `inner`. Such a common term doesn't mean much to my brain. I would almost rather see index being passed in here, but that's not what this reorg is about and everyone else might like it the way it is anyway. JTOL.